src/introduction/cross-ccc-release.md

```diff
@@ -7,7 +7,8 @@ using them in an ad hoc manner as we will do here is possible, but may take
 some getting used to.
 
 Begin by getting a shell in the docker container.  `cheribuild` can do this for
-you.  From the root of the unpacked release, run:
+you.  From the root directory of the unpacked release (`cheri/` by default),
```

Comment:
Maybe I was wrong to call this the "root of the unpacked release".  How about "From the `cheri/` directory of the unpacked release (which will be `~/cheri` if you have unpacked the release in your home directory), ..." or something like that?

Reply:
That sounds good for the workshop but for the upstream exercise book I'm OK with the term "root of the unpacked release" and the default value in parentheses.